playerbot/strategy/triggers/GenericTriggers.h

```diff
@@ -328,7 +328,10 @@ namespace ai
     class GreaterBuffOnPartyTrigger : public BuffOnPartyTrigger
     {
     public:
-        GreaterBuffOnPartyTrigger(PlayerbotAI* ai, string spell, string lowerSpell, int checkInterval = 2, bool ignoreTanks = false) : BuffOnPartyTrigger(ai, spell, checkInterval, ignoreTanks) {}
+        GreaterBuffOnPartyTrigger(PlayerbotAI* ai, string spell, string lowerSpell, int checkInterval = 2, bool ignoreTanks = false) : BuffOnPartyTrigger(ai, spell, checkInterval, ignoreTanks) 

```

Comment:
Write it like this to follow the coding standards please
`GreaterBuffOnPartyTrigger(PlayerbotAI* ai, string spell, string lowerSpell, int checkInterval = 2, bool ignoreTanks = false) : BuffOnPartyTrigger(ai, spell, checkInterval, ignoreTanks), lowerSpell(lowerSpell)`
and remove `this->lowerSpell = lowerSpell;`